src/options-schema.js

```diff
@@ -166,6 +189,9 @@ export default {
         archive: {
           $ref: '#/definitions/Archive',
         },
+        replace: {
```

Comment:
This `replace` action is very generic and might cause confusion? May be something like `ReplaceInFile` or something more explicit?

Thoughts?

Reply:
absolutely, let me make it better.

Reply:
hey @sibiraj-s, done!

I also updated the readme file.